api/src/models/User.ts

```diff
@@ -29,6 +39,50 @@ module.exports = (sequelize: Sequelize) => {
         defaultValue: DataTypes.UUIDV4,
         primaryKey: true,
       },
+      
+      UUID: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+      }, 
+
+      firstName: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+
+      lastName: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+      
+      username: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+
+      email: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+
+      mobile_zone: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+      mobile: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+
+      address: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+
+      imagenDePerfil: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
     },
     { timestamps: false }
```

Comment:
Todas las tablas tienen que ser tablas paranoicas (por el requerimeinto del borrador logico.)
```suggestion
      { 
        timestamps: false,
        paranoid: true,
      },
```